package.json

```diff
@@ -18,6 +18,9 @@
     "build/index.js",
     "build/example.js"
   ],
+  "dependencies": {
+      "prop-types": "^15.5.10"
```

Comment:
@stuwest How about we update this to lowest compatible version `^15.5.7` so more folks have a chance of de-duping the dep?

We'll also still want to bump the main package version from `1.1.0` to `2.0.0-alpha.1`  since we're jumping a peerDep from 14.x to 15.x and open a Calypso PR so we can test the change. We can either publish an alpha package with `npm publish --tag next`, where on the Calypso branch we install with `npm install gridicons@next`. OR we can point at the git url directly in package.json and the shinkwrap. The link for this branch would be `Automattic/gridicons#fix\/prop-types`

Before we merge this, we can set the version back to `2.0.0` and do a publish after it lands in master.